data/assets/scene/digitaluniverse/stars.asset

```diff
@@ -62,13 +62,7 @@ local Stars = {
   GUI = {
     Name = "Stars",
     Path = "/Milky Way/Stars",
-    Description = [[Census:  116,584 stars with 321 labels.<br> DU Version 8.10. This
-      star catalog is a combination of all available star catalogs, wherein we
-      choose the best distance available to place the stars around the Sun as
-      accurately as is possible. (Description from URL) <br><br> Data Reference:
-      Gaia Data Release 3 (Gaia Collaboration, 2022); XHIP: An Extended Hipparcos
-      Compilation (Anderson E., Francis C. 2012); Hipparcos Catalog (European Space Agency
-      1997); Gliese Catalog (Gliese and Jahriess 1991)]]
+    Description = [[These are the nearby stars that surround the Sun and are close enough to get accurate distances. These include all the stars we see with the unaided eye and many stars dimmer than that. Over the entire night sky, all year round, and in the northern and southern hemispheres, we can see roughly 9,000 stars total with the unaided eye. Stars are the light factories of the universe, and come in a variety of sizes, colors, and brightnesses. The base catalog is Hipparcos, with Gaia DR3 data applied for distance and velocity when available. Census: 112,746 stars.]]
```

Comment:
Line breaks have gone missing

Reply:
Oops. Thought I fixed all of them. Good catch